SETUP_NODE.md

```diff
@@ -0,0 +1,168 @@
+# Setup Hermez Node
+
+This tutorial will help you to setup an Hermez Node and run as sync mode.
+
+## Golang
+
+- [ ] [Install](https://golang.org/doc/install#install) Golang
+- [ ] [Create a](https://golang.org/doc/gopath_code#GOPATH) GOPATH environment
+- [ ] Set GOPATH/bin into PATH machine
+
+    ```command
+    export PATH=$PATH:$GOPATH/bin
+    ```
+
+## Go-Ethereum
+
+- [ ] Clone [GETH](https://github.com/ethereum/go-ethereum) project and do a `make all` command into project folder.
+  - Need have to gcc on PATH.
+
+    On Ubuntu/Debian you need install `build-essential`
+
+    ```command
+    sudo apt-get update
+    sudo apt-get install build-essential
+    ```
+
+- [ ] Add `build/bin` on PATH, after `make install` command inside of GETH
+folder.
+
+    ```command
+    export GETH_PATH=$GOPATH/src/github.com/ethereum/go-ethereum/
+    export PATH=$PATH:$GETH_PATH/build/bin
+    ```
+
+- [ ] Run the GETH
+
+    Enter in a empty folder (really) and you can run:
+
+    ```command
+    nohup geth --goerli --cache 4096 --http --http.addr INTERNAL_IP
+    --http.corsdomain *--http.vhosts* --http.api
+    "admin,eth,debug,miner,net,txpool,personal,web3" --ws --ws.addr INTERNAL_IP
+    --graphql --graphql.corsdomain *--graphql.vhosts* --vmdebug --metrics >
+    ~/temp/logs/geth-logs.log &
+    ```
+
+- [ ] Wait the geth command finished to move to next step.
+
+## PostgreSQL
+
+- [ ] [Install](https://www.postgresql.org/download/) PostgreSQL
+- [ ] Create a PostgreSQL User to project.
+
+    ```command
+    createuser --interactive
+    ```
+
+- [ ] Create a database
+
+    ```command
+    createdb hermez-db
+    ```
+
+- [ ] Create a password to User created at step 2
+
+    ```command
+    ALTER USER hermez WITH PASSWORD 'Your#Password';
+    ```
+
+- [ ] Test connection on localhost with all steps made until now
+
+    ```command
+    psql --host=localhost --dbname=hermez-db --username=db-username
+    ```
+
+- [ ] If you need expose the database:
+
+    In the `/etc/postgresql/<version>/main/postgresql.conf` file, change the value of `listen_address` to your host IP
+
+    ```command
+    listen_address = "1.2.3.4" #
+    ```
+
+## Nginx
+
+- [ ] [Install](https://www.nginx.com/resources/wiki/start/topics/tutorials/install/) Nginx
+
+- [ ] We need create configuration file to Hermez Node with this content:
+
+    File:
+
+    ```command
+    /etc/nginx/conf.d/hermeznode.conf
+    ```
+
+    Content:
+
+    ```txt
+    server {
+        listen 80;
+        listen [::]:80;
+
+        server_name EXTERNAL_IP;
+
+        location / {
+            proxy_pass http://localhost:8086/;
+        }
+    }
+    ```
+
+    This nginx configuration will help us to expose the hermez node.
```

Comment:
I believe we should add how to configure ssh here.

Reply:
IMHO the ssh configuration is a trivial thing to the developer. Maybe we should add a link to how to configure ssh?

Reply:
Sorry, typo here, not ssh, but https.